### ISSUE TITLE
Ooops no translation found

### ISSUE THREAD

**Issue by qwer1304** [artemave/translate_onhover#33]
Привет!
Иногда получаю "Ooops ... No translation found" на слово, которое Гугл прекрасно переводит.
Пример:
http://unsongbook.com/chapter-48-bring-me-my-chariot-of-fire/
Найди слово 'codger' и попробуй его перевести.

Запускаю перевод через Ctrl-Click.

Спасибо,
Д

**Comment by qwer1304** [comment]
Похоже, я нашел когда происходит эта ошибка:
1.Пойди на http://www.scottaaronson.com/blog/?p=3327
2.Найди слово "howlers"

Ошибка происходит:
3а.Кликни в середине слова
4.Ctrl-Click на слово - получишь ошибку

Ошибка НЕ происходит:
3б.Выбери слово (двойной щелчок)
4.Ctrl-Click на слово - НЕ получишь ошибку (получишь перевод)

**Comment by artemave** [comment]
У меня в обоих случаях ошибка. Однако если я выделяю howler (без s на
конце), то перевод работает.

2017-07-22 7:37 GMT+01:00 qwer1304 <notifications@github.com>:

**Comment by qwer1304** [comment]
Да, действительно, но если захбатить также и пробел после s в howlers, то ошибки нет.
(Раньше этого не заметил т.к. двойной щелчок захватыбает и пробел)

Можно это как-то исправить?
